neo/Core/FunctionCode.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, script=None, param_list=None, return_type=None, contract_prop
 
         if return_type and not isinstance(return_type, int) and len(return_type) > 1:
             return_type = return_type[0:1]
-        elif isinstance(return_type, int) and not 0 < return_type <= 255:
+        elif isinstance(return_type, int) and return_type > 255:
```

Comment:
just something I observed; `FunctionCode` is often initialized as `FunctionCode()` throughout the code base. This creates a return_type of `None`. There is no check that makes sure it becomes a valid return type. Perhaps we should make the default value to be `0xff` (void)?